Dockerfile

```diff
@@ -38,10 +37,7 @@ RUN echo " ... adding throttling support with ZMQ and CZMQ" \
                         --sysconfdir=/etc \
                         --mandir=/usr/share/man \
                         --infodir=/usr/share/info \
-         && make && make install \
-         && apk del automake autoconf \
-         && rm -rf /tmp/zeromq* && rm -rf /tmp/czmq* \
-         && rm -rf /var/cache/apk/*
+         && make && make install
```

Comment:
Sorry, this is my mistake, when reverting changes I made to test alpine:latest, I accidentally deleted the clean up part. Now is back on. Thanks